File: web3rpc/rpc-specs/code-samples/python/debug/logging/backtraceAt.py
from web3 import Web3
from web3py_ext import extend

host = "https://public-en-kairos.node.kaia.io"

location = "server.go:443"

w3 = Web3(Web3.HTTPProvider(host))
debug_response = w3.debug.backtrace_at(location)

print(debug_response)

File: web3rpc/rpc-specs/code-samples/javascript/debug/logging/backtraceAt.js
const { JsonRpcProvider } = require("@kaiachain/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-kairos.node.kaia.io");
  const location = "server.go:443";

  provider.debug
    .backtraceAt(location, {}, (err, data, response) => {})
    .then((data) => {
      console.log(data);
    });
})();
